### DISCUSSION TITLE
Styles.css is not loading after I link a custom domain

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#5128] [Bug Report]
### Page to Investigate

maximumdevmode.xyz

### Steps to Reproduce

Heyo! Before I linked my custom domain to Vercel, styles.css was showing and even getting applied to my files. As soon as I linked my custom domain and I visited the site, no css was applied or even found. Can anyone help?

Domain Registrar: Namecheap
Domains: maximumdevmode.xyz, www.maximumdevmode.xyz

**Comment by ghost** [comment]
**UPDATE!**

__I redeployed the site and now it is loading fine!__

**Reply by mikastamp** [reply]
Hey, I got the same issue, I tried to redeploy but it is still not working. What exact did you do? Would be greatful for help.

**Reply by LKonvalina** [reply]
Did you ever figure it out?
